docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM resin/rpi-raspbian:jessie
+
+ARG architecture=armv6
+ARG node_version=6.11.1
+ARG build_date
+ARG repo=redloro
+ARG branch=master
+
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.name="rpi-smartthings-nodeproxy" \
+      org.label-schema.description="SmartThings Node Proxy for Raspberry Pi" \
+      org.label-schema.version="1.0.2" \
+      org.label-schema.docker.cmd="docker run -d -p 8080:8080 -e ENABLED_PLUGINS='' --device=/dev/ttyUSB0 rpi-smartthings-nodeproxy" \
+      org.label-schema.build-date=$build_date \
+      architecture=$architecture
+
+RUN apt-get update \
+ && apt-get install wget \
+ && apt-get clean \
+ && wget -O - https://nodejs.org/dist/v${node_version}/node-v${node_version}-linux-${architecture}l.tar.xz \
+  | tar -xJvf - --strip-components=1 -C /usr/local/ \
+ && apt-get remove --auto-remove wget --purge \
+ && rm -rf /tmp/* 
+
+ENV NODE=/usr/local/bin/node
+ENV NPM=/usr/local/bin/npm
+ENV PYTHON=/usr/bin/python2.7
+
+RUN apt-get install python2.7 build-essential libpcap-dev wget \
+ && mkdir -p /stnp/plugins \
+ && wget -O - https://github.com/${repo}/smartthings/tarball/${branch} \
```

Comment:
@tcjennings @redloro Any reason you are pulling down a copy from GitHub rather than just using the current directory via Docker's `ADD` directive? It would make development much faster, as you wouldn't need to push to GitHub to test changes.